org.musescore.MuseScore.yml

```diff
@@ -1,4 +1,5 @@
 app-id:              org.musescore.MuseScore
+branch:              beta
```

Comment:
Flathub implies `stable`.

If you build it on your machine, it’s going to build as `master`, which is useful too, allowing you to install your local version alongside the Flathub one.

Flathub also requires `stable`, so we can’t do this. We also can’t ship beta-quality software on Flathub.

What you’re doing here is suggesting replacing the current stable MuseScore with the unstable beta one. There are cases where this makes sense, especially in old or poorly-maintained projects, but definitely not here.

The best solution would be to create a git branch and merge it when it’s ready.